include/zephyr/shell/shell.h

```diff
@@ -798,6 +799,9 @@ struct shell_ctx {
 	/** When bypass is set, all incoming data is passed to the callback. */
 	shell_bypass_cb_t bypass;
 
+	/*!< Logging level for a backend. */
```

Comment:
this should be: 

```suggestion
	/** Logging level for a backend. */
```

Any chance you could follow up with a commit to fix this? given this is API, it's always nicer if documentation is accurate there. Thanks!